src/components/EducationCard.tsx

```diff
@@ -30,7 +35,7 @@ function CompactGraphic({ title, icon, subtext, isSubtextContentLoading }: Compa
   return (
     <Grid container spacing={2}>
       <Grid item xs={12}>
-        <Typography variant="h6" align="center">
+        <Typography variant="h6" align="center" style={{ color: "#999999" }}>
```

Comment:
are you intentionally overriding the theme here?

Reply:
This is intentional. It's a shade of gray used for subtext/metrics as specified by the design team.

I'm going to instead use a pre-defined CSS class, to keep things simple.

Reply:
Done.